x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/configs/index.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useEffect, useState } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiCodeBlock,
+  EuiSpacer,
+  EuiSkeletonText,
+  EuiCallOut,
+  EuiLink,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import type { PackageInfo } from '../../../../../types';
+
+import { sendGetInputsTemplates, useStartServices } from '../../../../../hooks';
+
+interface ConfigsProps {
+  packageInfo: PackageInfo;
+}
+
+export const Configs: React.FC<ConfigsProps> = ({ packageInfo }) => {
+  const [configs, setConfigs] = useState<string | undefined>(undefined);
+  const [error, setError] = useState<undefined | Error>();
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const { notifications } = useStartServices();
+  const pkgName = packageInfo.name;
+  const pkgVersion = packageInfo.version;
+  // @ts-ignore-line
+  const notInstalled = !packageInfo?.installationInfo;
+
+  useEffect(() => {
+    const fetchYamlConfigs = async () => {
+      try {
+        const res = await sendGetInputsTemplates({ pkgName, pkgVersion }, { format: 'yml' });
+        setConfigs(res?.data as string);
+      } catch (e) {
+        setError(e);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+    fetchYamlConfigs();
+  }, [pkgName, pkgVersion]);
+
+  if (error) {
+    notifications.toasts.addError(error, {
+      title: i18n.translate('xpack.fleet.epm.InputTemplates.loadingErro', {
+        defaultMessage: 'Error input templates',
+      }),
+    });
+  }
+
+  return (
+    <EuiFlexGroup data-test-subj="epm.Configs" alignItems="flexStart">
+      <EuiFlexItem grow={1} />
+      <EuiFlexItem grow={6}>
+        {isLoading && !configs ? (
+          <EuiSkeletonText lines={10} />
+        ) : (
+          <>
+            <EuiText>
+              <p>
+                <FormattedMessage
+                  id="xpack.fleet.epm.InputTemplates.mainText"
+                  defaultMessage="View sample configurations for each of the {name} integration's data streams below. Copy/paste this YML into your {elasticAgentYml} file or into a file within your {inputsDir} directory. For more information, see the {userGuideLink}"
+                  values={{
+                    name: `${pkgName}`,
+                    elasticAgentYml: <code>elastic-agent.yml</code>,
+                    inputsDir: <code>inputs.d</code>,
+                    userGuideLink: (
+                      <EuiLink
+                        href="https://www.elastic.co/guide/en/fleet/current/elastic-agent-input-configuration.html"
```

Comment:
I think this should be added to the doc link service, so it will work for different version in the future